README.md

```diff
@@ -87,6 +87,11 @@ DocRaptor.download(download_key)
 
 The `download` function works like `DocRaptor.create` in that you get back either an HTTParty response object or you can give it a block.
 
+
+## Privacy
+
+To help us improve and support DocRaptor, this library will send DocRaptor information about the version of the gem you are using, as well as your ruby runtime. If you would like to not report that data, you can do so by calling `DocRaptor.disable_agent_tracking` when you initialize the library.
```

Comment:
I think it would be worth it to be even more explicit in this part - include an example user agent string, so that they can see exactly what is being sent.

Maybe  a second sentence like "For example, as of the time of writing this documentation, my reported user agent string is 'expectedbehavior_doc_raptor_gem/0.4.3 ruby/2.2.2'." or something like that.

Not that it's hard to find exactly what is being sent.